ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -23,14 +23,15 @@
 '''
 
 LOOKAHEAD_WPS = 200 # Number of waypoints we will publish. You can change this number
-
+MAX_DECEL = 10
```

Comment:
MAX_DECEL in another module (`waypoint_loader.py`) is set to `1.0`.

Reply:
alright, this is the right value to use as corroborated by dbw.launch file in twist controller package

```
<?xml version="1.0"?>
<launch>
    <node pkg="twist_controller" type="dbw_node.py" name="dbw_node">
        <param name="vehicle_mass" value="1736.35" />
        <param name="fuel_capacity" value="13.5" />
        <param name="brake_deadband" value=".1" />
        <param name="decel_limit" value="-1." />
        <param name="accel_limit" value="1." />
        <param name="wheel_radius" value="0.2413" />
        <param name="wheel_base" value="2.8498" />
        <param name="steer_ratio" value="14.8" />
        <param name="max_lat_accel" value="3." />
        <param name="max_steer_angle" value="8." />
    </node>
</launch>

```